tests/test_filters.py

```diff
@@ -332,20 +333,26 @@ def testFilterSigns(self):
         self.assertTrue(np.allclose(sign_filter.evaluate(self.omega), signs))
 
 
-@pytest.mark.parametrize("dtype", [np.float32, np.float64])
-def test_power_filter_safeguard(dtype, caplog):
+params = list(itertools.product([np.float32, np.float64], [None, 0.01]))
```

Comment:
Don't we do this with fixtures these days? Might as well update.

Reply:
Good point. Done. Thanks!

Reply:
Nice. Thank you!